cirq/api/google/v2/util_test.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+import cirq
+import cirq.api.google.v2.util as util
+
+
+@pytest.mark.parametrize('sweep', [
+    cirq.UnitSweep,
+    cirq.Linspace('a', 0, 10, 100),
+    cirq.Points('b', [1, 1.5, 2, 2.5, 3]),
+    cirq.Linspace('a', 0, 1, 5) * cirq.Linspace('b', 0, 1, 5),
+    cirq.Points('a', [1, 2, 3]) + cirq.Linspace('b', 0, 1, 3),
+    ])
+def test_sweep_to_proto(sweep):
+    msg = util.sweep_to_proto(sweep)
+    deserialized = util.sweep_from_proto(msg)
+    assert deserialized == sweep
```

Comment:
Add at least one test that verifies an intermediate form is actually a protobuf with the right content. I've seen these kind of round-trip-only tests let nasty bugs through in the past.

Reply:
done.